controllers/suggestions/suggest_offseason_event_review_controller.py

```diff
@@ -0,0 +1,102 @@
+from datetime import datetime
+
+from consts.account_permissions import AccountPermissions
+from consts.district_type import DistrictType
+from consts.event_type import EventType
+from controllers.suggestions.suggestions_review_base_controller import \
+    SuggestionsReviewBaseController
+from helpers.event_manipulator import EventManipulator
+from models.event import Event
+from models.suggestion import Suggestion
+from template_engine import jinja2_engine
+
+
+class SuggestOffseasonEventReviewController(SuggestionsReviewBaseController):
+
+    def __init__(self, *args, **kw):
+        self.REQUIRED_PERMISSIONS.append(AccountPermissions.REVIEW_OFFSEASON_EVENTS)
```

Comment:
SuggestionsReviewBaseController doesn't make sense now, since we changed MUTATE_DATA to REVIEW_MEDIA. perhaps it's best to make `REQUIRED_PERMISSIONS = []` in the base controller, and follow this new pattern for the media review controllers.